docs/modules/ROOT/pages/glossary/index.adoc

```diff
@@ -101,5 +98,8 @@ A mechanism to secure the software supply chain by recording events in a user-de
 **Tekton integration testing** +
 A process that uses Tekton tasks to support the setup and execution of dynamic application tests against container images.
 
+**Tekton task** +
+One or more steps that run container images. Each container image performs a piece of construction work.
+
 **workspace** +
-A storage volume that a task requires at runtime to receive input or provide output.
+A technology framework that brings together all of the applications, tools, and resources teammates need to work and collaborate securely, while unifying and enhancing application delivery and access control.
```

Comment:
Do you want to specify this to {ProductName} by something like 

> A technology framework in {ProductName} that [...]

Reply:
well, my thinking is that this will be contained in a glossary that's RHTAP-specific, so the prod name is implied. wdyt?

Reply:
Is a glossary strictly saying what the thing is? or should it provide context in the user's world?
Every user who signs up gets their own workspace. Users can be invited with specific permissions (see: Roles) to collaborate in other workspaces (such a product team's workspace). There are also managed workspaces, where teams can configure their application to release to.